src/Nri/Colors.elm

```diff
@@ -166,6 +167,16 @@ coralLightest =
 
 {-|
 
+<p style="font-size:2em; color: #eef9ff; background-color: black;">#eef9ff</p>
```

Comment:
:thought_balloon: This might be a good place for elm-verify-example to make sure the colors match the actual value